From 28a5f6f00f1dbf961554ac2a8ae2607ead92af9c Mon Sep 17 00:00:00 2001
From: Tang Yuantian <Yuantian.Tang@nxp.com>
Date: Wed, 27 Jul 2016 13:45:01 +0800
Subject: [PATCH 011/124] dts: ls2088: remove ls1021a compatible string from
 sata node

ls1021a is not compatible with ls2088a as for sata, so remove it
from sata nodes.

Signed-off-by: Tang Yuantian <yuantian.tang@nxp.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls2088a.dtsi | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls2088a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls2088a.dtsi
index fbb52fe..892d426 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls2088a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls2088a.dtsi
@@ -597,8 +597,7 @@
 
 	sata0: sata@3200000 {
 		status = "disabled";
-		compatible = "fsl,ls2088a-ahci", "fsl,ls2080a-ahci",
-			     "fsl,ls1021a-ahci";
+		compatible = "fsl,ls2088a-ahci", "fsl,ls2080a-ahci";
 		reg = <0x0 0x3200000 0x0 0x10000>;
 		interrupts = <0 133 0x4>; /* Level high type */
 		clocks = <&clockgen 4 3>;
@@ -606,8 +605,7 @@
 
 	sata1: sata@3210000 {
 		status = "disabled";
-		compatible = "fsl,ls2080a-ahci", "fsl,ls2080a-ahci",
-			     "fsl,ls1021a-ahci";
+		compatible = "fsl,ls2088a-ahci", "fsl,ls2080a-ahci";
 		reg = <0x0 0x3210000 0x0 0x10000>;
 		interrupts = <0 136 0x4>; /* Level high type */
 		clocks = <&clockgen 4 3>;
-- 
2.1.0.27.g96db324

